apps/service-portal/src/components/Sidebar/Sidebar.css.ts

```diff
@@ -2,18 +2,49 @@ import { style } from '@vanilla-extract/css'
 import { theme, themeUtils } from '@island.is/island-ui/theme'
 
 export const sidebar = style({
+  position: 'absolute',
+  top: 0,
+  left: 0,
+  zIndex: 10,
   height: '100%',
-  minWidth: 250,
   marginBottom: theme.spacing['10'],
+  transition: 'all 250ms ease-in-out',
+  width: 252,
+  backgroundColor: theme.color.blue100,
+})
+
+export const collapsed = style({
+  width: 138,
+})
+
+export const logoCollapsed = style({
+  display: 'flex',
+  justifyContent: 'flex-end',
+  marginRight: -22,
+})
+export const subnav = style({
+  paddingLeft: 20,
+  marginLeft: 85,
+  borderLeft: `1px solid ${theme.color.blue200}`,
   ...themeUtils.responsiveStyle({
-    lg: {
-      width: 306,
+    md: {
+      paddingLeft: 26,
+      marginLeft: 37,
```

Comment:
Why are we not following the spacings from theme for this?

This might look inconsistent.

Reply:
fixed